fvm/handler/event.go

```diff
@@ -140,7 +141,8 @@ func (e *EventCollection) TotalByteSize() uint64 {
 // TODO refactor this
 // Keep serviceEventWhitelist module-only to prevent accidental modifications
 var serviceEventWhitelist = map[string]struct{}{
-	"EpochManager.EpochSetup": {},
+	"FlowEpoch.EpochSetup":     {},
+	"FlowEpoch.EpochCommitted": {},
```

Comment:
note about rename

Reply:
Need to refactor to include service account prefix

Reply:
> Need to refactor to include service account prefix

I don't think this is necessary, see above comment

Reply:
Name change captured in https://github.com/dapperlabs/flow-go/issues/5593